valgrind.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# cd to the script directory:
+cd "${0%/*}" || { echo "Couldn't cd to ${0%/*}!"; exit 1; }
+VALGRIND="valgrind"
+if ! [ -x "$(command -v ${VALGRIND})" ]; then
+  echo "{$VALGRIND} not found. skipping..."
+  exit 1
```

Comment:
If you exit 1, it's not skipping :-)

Reply:
Yes, it does.
I borrowed this code from clang-format.sh

Reply:
Maybe we are not using "skipping" with the same meaning.
Do you mean the script should return 0 if valgrind is not found?

Reply:
"Skipping" suggests to me that it's not failing, but "exit 1" is failure.

It really matters little though.